### ISSUE TITLE
Improve usability of long lists

### ISSUE THREAD

**Issue by dantefromhell** [humhub/humhub#5784]
The current example describes the problem when trying to select the country in a profile, but I've also seen similar issues with the emjoii selector.

On HumHub 1.11.2 trying to choose the country in the profile settings opens this dialog.
![image](https://user-images.githubusercontent.com/90508808/177001937-db1f3793-4b9e-48d9-af0b-b6bd2be31c7d.png)

I've seen user struggle with this dialog in the following ways
1) The textbox [1] to search for a country is almost invisible. It took me 2min of staring at the screen to actually see it. One aspect that IMO increases this issue is that the list is the furthest away from the mouse cursor.
2) The typical behaviour of clicking the drop-down list and starting to type for searching is not working because the textbox [1] does not have focus set automatically. I also observed this issue with the emojii chooser.

I believe the behaviour could be greatly improved with
- Automatically setting the text cursor into the textbox [1]. 
- Moving textbox [1] to be rendered at the bottom of the list and make it visually more outstanding in the default theme.

**Comment by ArchBlood** [comment]
I agree, the use of `auto-focus` should be used in fields that require text input, normally the browser does this automatically but it works much different for HumHub for some reason.
